plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -235,16 +253,14 @@ export class EventsProcessor {
             created_at: castTimestampOrNow(null, timestampFormat),
         }
 
-        let eventId: Event['id'] | undefined
-
         const useExternalSchemas = this.clickhouseExternalSchemasEnabled(teamId)
         // proto ingestion is deprecated and we won't support new additions to the schema
         const message = useExternalSchemas
             ? (EventProto.encodeDelimited(EventProto.create(eventPayload)).finish() as Buffer)
             : Buffer.from(
                   JSON.stringify({
                       ...eventPayload,
-                      person_id: personInfo?.uuid,
+                      person_id: eventPersonUuid,
```

Comment:
Q to @yakkomajuri - this is changing from undefined -> null if person could not be found, but person properties was previously null. Thoughts on handling bottom types here?